apps/settings/js/panels/bluetooth/panel.js

```diff
@@ -193,6 +195,8 @@ define(function(require) {
           BtContext.stopDiscovery();
         }
 
+        BtContext.unobserve('discoverable', this._boundUpdateVisibilityTimer);
```

Comment:
I said un-observe the handler in `hide` just means that we should un-register it when leaving the panel. If you already un-observe other funcitons in `onBeforeHide`, let's do it there in one place.